src/utils/requestUtils.ts

```diff
@@ -48,4 +50,27 @@ export namespace requestUtils {
             stream.pipe(fse.createWriteStream(filePath).on('finish', resolve).on('error', reject));
         });
     }
+
+    // tslint:disable-next-line:no-any
+    export async function getFunctionAppMasterKey(resourceId: string, actionContext: IActionContext): Promise<WebSiteManagementModels.HostKeys | undefined> {
```

Comment:
This function isn't really related to `requestUtils` anymore. Just use the two lines directly instead of creating a function here